src/Fabulous.XamarinForms/Views/Controls/Button.fs

```diff
@@ -4,23 +4,80 @@ open System.Runtime.CompilerServices
 open Fabulous
 open Xamarin.Forms
 
+[<Struct>]
+type ImagePosition =
+    | Left
+    | Top
+    | Right
+    | Bottom
+
+[<Struct>]
+type ContentLayout =
+    | ContentLayout of position: ImagePosition * spacing: float
+    member this.Position =
+        this
+        |> fun (ContentLayout (position, _)) -> position
+
+    member this.Spacing =
+        this
+        |> fun (ContentLayout (_, spacing)) -> spacing
+
 type IButton =
     inherit IView
 
 module Button =
     let WidgetKey = Widgets.register<Button> ()
 
+    let BorderColor =
+        Attributes.defineAppThemeBindable<Color> Button.BorderColorProperty
+
+    let BorderWidth =
+        Attributes.defineBindable<float> Button.BorderWidthProperty
+
+    let CharacterSpacing =
+        Attributes.defineBindable<float> Button.CharacterSpacingProperty
+
+    let ContentLayout =
+        Attributes.defineBindable<Xamarin.Forms.Button.ButtonContentLayout> Button.ContentLayoutProperty
+
+    let CornerRadius =
+        Attributes.defineBindable<int> Button.CornerRadiusProperty
+
+    let FontAttributes =
+        Attributes.defineBindable<Xamarin.Forms.FontAttributes> Button.FontAttributesProperty
+
+    let FontFamily =
+        Attributes.defineBindable<string> Button.FontFamilyProperty
+
+    let FontSize =
+        Attributes.defineBindable<float> Button.FontSizeProperty
+
+    let ImageSource =
+        Attributes.defineBindable<Xamarin.Forms.ImageSource> Button.ImageSourceProperty
+
+    let ImageSource' =
+        Attributes.defineAppThemeBindable<string> Button.ImageSourceProperty
```

Comment:
You should not declare 2 attributes for the same property.
See `Image.fs` for how to instantiate a same property with several types.

Reply:
I realised that we only need one TBH the once that accepts light and dark images as this will cover both extension methods